public/data/food/ingredients.json

```diff
@@ -168,7 +168,7 @@
     "id": "flour-organic",
     "name": "Farine bio (sortie de moulin)",
     "categories": ["grain_processed", "organic"],
-    "default": "850ecaa9-7cfe-da68-5ae5-ecf99d08fed3",
+    "default": "504bb4e8-f405-056a-4c90-ad15abba42cf",
```

Comment:
Are we ok with the default ID (the foreign key) changing here and in the processes.json? Is it going to change again in the future? Why is it changing? Should we instead have a fixed ID like the ones that were created like `AGRIBALU...`?

Reply:
I think it's due to the naming change of constructed processes. It was based on the search term, I've replaced it with the real name of the activity.  I'll try to find a way to have a more reproducible ids.